README.md

```diff
@@ -10,7 +10,7 @@ IPTV Live TV and Radio PVR client addon for [Kodi](http://kodi.tv)
 ### Linux
 
 1. `git clone --branch Leia https://github.com/xbmc/xbmc.git`
-2. `git clone https://github.com/kodi-pvr/pvr.iptvsimple.git`
+2. `git clone --branch Leia https://github.com/kodi-pvr/pvr.iptvsimple.git`
```

Comment:
Unrelated change. Should be in separate commit.

Reply:
ok